File: week04-2.py
#week04-2.py是今天的LeetCode挑戰Easy題(二合一)
#Maximum Count of Positive Integers and Negative Integers
#找出 有幾個正數 有幾個負數 大的那個回傳
class Solution:
    def maximumCount(self, nums: List[int]) -> int:
        pos=0
        neg=0#迴圈前面，準備好，都放0
        for i in range(len(nums)):#迴圈裡面，更新他
            if nums[i] > 0 : pos+=1
            if nums[i] < 0 : neg+=1
        #迴圈後面把答案拿來用
        return max(pos,neg)

File: week04-2.c
//week04-2.py琌さぱLeetCode珼驹Easy肈()
//Maximum Count of Positive Integers and Negative Integers
//т Τ碭タ计 Τ碭璽计 ê肚
int maximumCount(int* nums, int numsSize) {
    int pos=0, neg=0;//癹伴玡常琌0
    for(int i=0;i<numsSize;i++){
        if(nums[i]>0)pos++;//タ计
        if(nums[i]<0)neg++;//璽计
    }//癹伴い丁穝э

    //癹伴рウㄓノ
    if(pos>neg) return pos;//タ计碞タ计
    else return neg;//ぃ礛碞璽计
}
